src/builder/Actions.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import {CCTP} from "./BridgeRoutes.sol";
+import {Strings} from "./Strings.sol";
+import {Accounts} from "./Accounts.sol";
+import {CodeJarHelper} from "./CodeJarHelper.sol";
+
+import {TransferActions} from "../DeFiScripts.sol";
+
+import {IQuarkWallet} from "quark-core/src/interfaces/IQuarkWallet.sol";
+
+library Actions {
+    string constant PAYMENT_METHOD_OFFCHAIN = "OFFCHAIN";
+    string constant PAYMENT_METHOD_PAYCALL = "PAY_CALL";
+    string constant PAYMENT_METHOD_QUOTECALL = "QUOTE_CALL";
+
+    string constant ACTION_TYPE_BRIDGE = "BRIDGE";
+    string constant ACTION_TYPE_TRANSFER = "TRANSFER";
+
+    // With Action, we try to define fields that are as 1:1 as possible with
+    // the simulate endpoint request schema.
+    struct Action {
+        uint256 chainId;
+        address quarkAccount;
+        string actionType;
+        bytes actionContext;
+        // One of the PAYMENT_METHOD_* constants.
+        string paymentMethod;
```

Comment:
I guess we should put a `TODO:` for paymentMethod?
It seems I can't find where it gets initialized or specified anywhere 🤔

Reply:
This is set in #8